conn/physic/example_test.go

```diff
@@ -67,7 +67,7 @@ func ExampleForce() {
 	fmt.Println(physic.PoundForce)
 	// Output:
 	// 10mN
-	// 990.569kN
+	// 990.570kN
```

Comment:
I really wonder why I did the multiplication. I think you can replace line 66 with `fmt.Println(physic.EarthGravity)`